src/reducers/ChatReducer.ts

```diff
@@ -37,6 +37,11 @@ export const hasLeftChatAction = createAction('HAS_LEFT_CHAT', cb => {
   return (roomID: string, name: string) => cb({ roomID, name });
 });
 
+export const joinChatAction = createAction('JOIN_CHAT', cb => {
```

Comment:
```suggestion
export const joinChatAction = createAction('JOIN_CHAT', callback => {
```

Kan være greit å endre til callback istedenfor cb for bedre lesbarhet for de som ser kodebasen for første gang